client/signup/steps/about/index.jsx

```diff
@@ -543,14 +543,10 @@ export default connect(
 		isLoggedIn: isUserLoggedIn( state ),
 		verticalId: getSiteVerticalId( state ),
 		verticalParentId: getSiteVerticalParentId( state ),
-		shouldHideSiteGoals:
-			'onboarding' === ownProps.flowName && includes( ownProps.steps, 'site-type' ),
 		shouldHideSiteTitle:
 			'onboarding' === ownProps.flowName && includes( ownProps.steps, 'site-title' ),
 		shouldSkipAboutStep:
-			includes( ownProps.steps, 'site-type' ) &&
```

Comment:
Now `site-type` step is no longer used. We only use `site-type-with-theme` step.

Reply:
You're right! `site-topic` step is no longer in use. But I think we can remove this property as it is true only when the steps contain all of the `site-type`, `site-topic` and `site-title`.

Reply:
I've also removed `shouldHideSiteTitle` and `shouldSkipAboutStep`!